README.md

```diff
@@ -9,6 +9,7 @@ A Modern MP3 Player
 $ git clone https://github.com/sanket143/MJam.git
 $ cd MJam
 $ npm install
+$ npm install -g electron
```

Comment:
We don't need to install electron explicitly as it is mentioned in the dependencies in `package.json`.